## On the road to breeding 4.0
[Wallace et. al. 2018](https://www.annualreviews.org/doi/pdf/10.1146/annurev-genet-120116-024846)
- Some traits are discrete but most are quantitative in nature.
- Although quantitative genetics is century old, its principle have been applied throughout the history of plant breeding.

#### Four stages of breeding:
- 1.0: 
    Method: Phenotype based selection
        
    Triggered by: Domestication and rise of modern human civilization and begining of agricultural practices
        
    Timeline: began 10-12 kYA
        
- 2.0: start of the science of breeding (Rise of Fisher, and quantitative concept of breeding)
    Method: replicated trails, controlled design and crosses, statistical and experimental designs, hybrid vigor, pedigree-breeding and precise measurements using tools/scales.
         
    Triggered by: Inbreeding depression, Mendelian genetics, development of statistics and quantitative and pop. genetics
         
    Timeline: began towards the end of 19th and begining of 20th century.
         
- 3.0: genomic data started complementing phenotypic data
    Method: Marker-assisted backcrossing/selection, genetic mapping, genomic selection, high throughput genotyping
         
    Triggered by: molecular markers, next-generation sequencing, computational tools and exploiting natural variation
         
    Timeline: From late 1980s.
         
- 4.0:
New level of breeding where functional genetic variants can be rationally combined both faster and better than before. Catalyzed by rapidly advancing genetics and information systems technologies. HTP can measure numerous traits longitudinally across the growth phase of plants and machine learning can process and interpret agronomic data beyond human's capabilities. Genome editing is expected to enable parallel precise modifications at many sites per generation.
         
    One could envision 5.0 involving de novo design of genes, pathways and traits, but it is far in future.
      
 
 #### Three key  questions to consider:
 a. How do we adapt crops to better fit agricultural environments?
 
Crops didn't evolve in agricultural settings, modern agriculture put different selective pressures on plant than was present in their ancestors. No crop has reached it's [adaptive peak/maximum fitness](https://en.wikipedia.org/wiki/Fitness_landscape#In_evolutionary_optimization), and finding ways to move towards the peak is the ultimate goal of plant breeding.

Three main aspects where increased understanding is necessary to advance breeding:
  
 - New Mega-Environments: Crops originated locally and were dispersed [khoury et. al.](https://royalsocietypublishing.org/doi/pdf/10.1098/rspb.2016.0792), and due to monoculture the crops have adjusted to new mega-environments since domestication; understanding adaptation, parsing out adaptation from population structure.
 - Interplant Competition: Homgeneous vs heterogenous environment, Competition vs Co-existence; early selection methods in breeding favors competition but how to select for the maximum productive lines under homogeneous environment, [group selection](https://en.wikipedia.org/wiki/Group_selection), mapping competetiveness of plants and incorporating that in selection [Prognostic breeding](https://onlinelibrary.wiley.com/doi/10.1002/9781118497869.ch6), [in-silico modeling](https://onlinelibrary.wiley.com/doi/epdf/10.1111/pce.12673).
     - Soil Interactions: soil is one of the most complex ecosystems [metagenomics of soil](https://www.nature.com/articles/nrmicro1160)
     microbial communities can alter host phenotypes although very little is known about the quantitative genetics involved.
     Rhizosphere/Environment is the biggest driver of plant microbial communities, so it might be easier to adjust management
     practices or apply microbes as seed treatment than breed for improved associations.
     
b. What is the nature of the diversity upon which breeding can act?

Although breeders can treat genetics like black box, understanding the nature of genetic variation improves breeding efforts.
Comparative studies can therefore give clues about adaptation and important genetic variation to improve adaptation.
Major conclusions of WG comparisions is that variation in gene content and copy number are the largest drivers of plant variation rather than differences in protein sequences.
Not all copy number variations are important though.

- Polyploidy:
[subgenome dominance](http://www.plantcell.org/content/29/9/2150),  [Jiao  et. al., nature 2011](https://www.nature.com/articles/nature09916.pdf). most polyploidy events are evolutionary dead ends.
[Wendel JF, essay](https://bsapubs.onlinelibrary.wiley.com/doi/full/10.3732/ajb.1500320): Microevolutionary and macroevolutionary events after WGD, very little is understood about events that tie the two together. How does microevolutionary mechanisms ultimately lead to ecological, physiological, and morphological variations?
  
- Tandem duplication:
results from replication errors or unequal crossing over during meiosis.
Since they are under effect of native regulatory elements, they provide easy mechanism for modification of gene dosage.
Example: [Caretenoid degradation in maize](https://www.ncbi.nlm.nih.gov/pubmed/28159756), dwarfing in wheat and sorghum, Al tolerance in maize, salt tolerance in wheat.

- The size of mutational space:
How many mutations can shift a crop towards the same goal?
Various mutations in the same gene are usually responsible for multiple manifestations of phenotypic variations.
In practice, there are only limited number of routes to alter a given phenotype.
Eventhough the distance between DNA elements can vary dramatically across plant genomes, the  actual genomic space that is important seems to be much smaller and constant.
Functional mutational space of plants might be quite small.
 
c. How do we deal with deleterious variants?
Breeders don't think much about deleterious mutations. most of the deleterious mutation are eliminated earlier and milder mutations go unseen without giving much thought to the mechanisms.
Identifying, controlling, and repairing these mutations is likely to be a major aspect of research as we move to Breeding 4.0.

- Deleterious allele and domestication:
Population genetic theory explaination of increase in detrimental alleles due to domestication:
    1. Domestication alters selective pressures so that traits favored in the wild become neutral or disfavored under
        domestication.
    2. Purifying selection is frequently reduced after domestication bottlenecks.
    3. Inbreeding of domesticated varieties further reduces their effective population size and effect recombination rates.

Emerging evidences suggests that this cost of domestication can be tempered by modern improvement practices. Ex: reduced genetic load in modern inbred lines of maize compared to outcrossing landraces.

- The problem with chromosomes :
Problem with deleterious mutations is that biology of chromosomes makes it difficult to breed them out.
A haplotype has mix of good and bad alleles due to linkage, and recombination is often not efficient enought to ever create a single ideal combination. Instead, several suboptimal haplotypes selectively interfere and none reach fixation, this phenomenon is dubbed [Hill-Roberston interference/effect](https://www.ncbi.nlm.nih.gov/pubmed/5980116) and results in negative LD (repulsion phase) between beneficial variants in low-recombination regions.

   Consequences of Hill-Robertson effect in plant breeding:
    - Interference reduces efficacy of selection on any individual site.
    - Having beneficial alleles spread across multiple nonrecombining haplotypes reduces genetic variance by eliminating
        extreme phenotypes, which in turn decreases raw material for selection to act upon.
    - Assuming most deleterious alleles are at least partly recessive, the low recombination regions where interference is most
        severe should benefit more from heterozygosity than high-recombination regions.

- Implications for Heterosis
     
 
     
